src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import './Tabs.scss';
+
+export const Tabs = (props) => {
+  const { tabs, togglerHandler, indexOfActive } = props;
+
+  return (
+    <div className="Tabs">
+      <div className="nav nav-tabs">
+        {tabs.map((elem, index) => (
+          <li className="nav-item">
+            <a
+              href="#linterPlug"
+              type="button"
+              key={elem.title.split(' ').join('')} // word without spaces
```

Comment:
what is the problem with spaces?

Reply:
it is bad key with spaces I think

Reply:
I can change back, but as for me this is better

Reply:
I don't see any problems in spaces for key, it just have to be unique

Reply:
FYI ⬆️
I'm approving